README.md

```diff
@@ -48,16 +48,16 @@ Download and install Python `v3.10.10`
     pip install -r requirements.txt
     ```
 
-3. From the `src` directory, run the following command to run the app
+3. From the `app` directory, run the following command to run the app
     ```bash
-    python3 -m uvicorn bananabot:app
+    python3 -m uvicorn main:app
     ```
 
 ## Usage
 
-1. Set the `url` class variable in [src/announcement.py](./src/announcement.py) to your [webhook](https://api.slack.com/messaging/webhooks)
+1. Set the `url` class variable in [app/models/app_config.py](./app/models/app_config.py) to your [webhook](https://api.slack.com/messaging/webhooks). (Note this is hard coded intentionally to avoid the potential for misuse)
```

Comment:
```suggestion
1. Set the `url` class variable in [app/models/app_state.py](./app/models/app_state.py) to your [webhook](https://api.slack.com/messaging/webhooks). (Note this is hard coded intentionally to avoid the potential for misuse)
```